automation/devops_automation_infra/k8s_plugins/kubectl.py

```diff
@@ -25,13 +26,24 @@ def _tunnel(self):
         tunnel = self._master.TunnelManager.get_or_create('kubectl', svc_ip, svc_port, ssh.get_transport())
         return tunnel
 
+    def _is_k3d(self):
+        ssh = self._master.SshDirect
+        try:
+            ssh.execute("docker ps | grep k3s > /dev/null 2>&1")
```

Comment:
k3d or k3s?

Reply:
k3s is part of k3d node entrypoint. That's why it works that way